noxfiles/utils_nox.py

```diff
@@ -25,13 +25,13 @@
 @nox.session()
 def create_user(session: nox.Session) -> None:
     """Create a super user in the fidesops database."""
-    run_infrastructure(datastores=["postgres"], run_create_superuser=True)
+    run_infrastructure(datastores=[], run_create_superuser=True)
 
 
 @nox.session()
 def seed_test_data(session: nox.Session) -> None:
     """Seed test data in the Postgres application database."""
-    run_infrastructure(datastores=["postgres"], run_create_test_data=True)
+    run_infrastructure(datastores=[], run_create_test_data=True)
```

Comment:
I noticed we can execute the infra commands like this now, allowing us to retain existing functionality but explicitly specify if we don't want any datastores to come up.